packages/integrations/useFocusTrap/index.ts

```diff
@@ -113,7 +110,10 @@ export function useFocusTrap(target: MaybeElementRef, options: UseFocusTrapOptio
     }, { flush: 'post' })
 
   // Cleanup on unmount
-  tryOnUnmounted(deactivate)
+  tryOnUnmounted(() => {
+    stop()
```

Comment:
No need for stopping a watch with this hook - it's called automatically.

Reply:
Ah right, forgot 😅 

Fixed